packages/database/scripts/setupTestDatabase.sh

```diff
@@ -14,6 +14,7 @@ fi
 
 PGPASSWORD=$DB_PG_PASSWORD psql -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "DROP DATABASE IF EXISTS $DB_NAME"
 PGPASSWORD=$DB_PG_PASSWORD psql -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "CREATE DATABASE $DB_NAME WITH OWNER $DB_USER"
+PGPASSWORD=$DB_PG_PASSWORD psql -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "CREATE EXTENSION postgis"
```

Comment:
I think we need to do a check to see if postgis already exists. Should be doable using something like:
```
SELECT * FROM pg_extension WHERE extname = 'postgis';
```